docs/ramalama-serve.1.md

```diff
@@ -111,6 +111,10 @@ use. Using this option RamaLama will override these defaults.
 On Nvidia based GPU systems, RamaLama defaults to using the
 `nvidia-container-runtime`. Use this option to override this selection.
 
+#### **--output-dir**=path
```

Comment:
Not crazy about options with - in them. --output or --out-dir?

Should throw an error when using --output-dir without --generate.

Rather then just adding another option, what do you think about adding :PATH to --generate option?  --generate quadlet:dest=/tmp/myquadlets?

Reply:
Adding a :PATH to the generate option seems good to me. The output directory only makes sense with this specific option. By default, :PATH is not specified and defaults to the current directory from where ramalama is executed (as it currently is).

Reply:
Pushed an update to extend the `--generate` with a path, e.g. `--generate quadlet:/etc/containers/systemd`.
PTAL @rhatdan @ericcurtin